web/src/main/java/nl/tehdreamteam/se42/web/command/CommandHandler.java

```diff
@@ -0,0 +1,121 @@
+package nl.tehdreamteam.se42.web.command;
+
+import nl.tehdreamteam.se42.web.command.impl.HelpCommand;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Handles {@link Command Commands} registered in this {@code CommandHandler}.
+ */
+public class CommandHandler {
+
+    private static final String COMMAND_ARGUMENT_SPLIT_TOKEN = " ";
+
+    private final Map<String, Command> commands = new HashMap<>();
+
+    public CommandHandler() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Document this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)